guides/v2.1/architecture/archi_perspectives/persist_layer.md

```diff
@@ -14,14 +14,25 @@ If you expect to return multiple items from a database query, then you would imp
 
 A simple resource model defines and interacts with a single table.
 
-However, some objects have a vast number of attributes, or they could have a set related objects that have varying numbers of attributes. In these cases, the objects are constructed using **Entity-Attribute-Value (EAV)** models. 
+However, some objects have a vast number of attributes, or they could have a set related objects that have varying numbers of attributes. In these cases, the objects are constructed using **Entity-Attribute-Value (EAV)** models.
 
 Any model that uses an EAV resource has its attributes spread out over a number of MySQL tables.
 
 The `Customer`, `Catalog` and `Order` resource models use EAV attributes.
 
+## XML Declaritive schema
+
+With Magento 2.3, we introduced Declaritive XML Schemas.
+These are XML files that are used to specify the final state of the database.
```

Comment:
Consider combining these two sentences into something like this:
With Magento 2.3, we introduced Declaritive XML Schemas, XML files used to specify the final state of the database.